specification/cat/_types/CatBase.ts

```diff
@@ -28,3 +28,126 @@ import { RequestBase } from '@_types/Base'
 export class CatRequestBase
   extends RequestBase
   implements CommonCatQueryParameters {}
+
+export enum CatAnonalyDetectorColumn {
+  /** @aliases ae */
+  assignment_explanation = 0,
+  /** @aliases bc, bucketsCount */
+  buckets.count = 1,
```

Comment:
You can't use a dot without wrapping the name inside quotes.
```suggestion
  'buckets.count' = 1,
```

Reply:
Thanks!